profiles/databricks.yaml

```diff
@@ -3,6 +3,11 @@
 # experiment:
   # name: "/Shared/sklearn_regression_experiment"
 
+# Set the registry server URI. This property is especially useful if you have a registry
+# server that’s different from the tracking server.
+# model_registry:
+#   uri: sqlite:////tmp/registry.db
```

Comment:
The example given here does not conform to Databricks registry URL. This example is more for `local.yaml`. Can we update the example with a real databricks shard URL? We should also test that URL actually works in cross workspace access scenario.